lib/slack-ruby-bot/rspec/support/fixtures/slack/migration_in_progress.yml

```diff
@@ -28,3 +28,31 @@ http_interactions:
       string: '{"ok":false,"error":"unknown"}'
     http_version:
   recorded_at: Tue, 28 Apr 2015 12:55:22 GMT
+- request:
```

Comment:
You have a spec that's making 3 requests somehow, this shouldn't be here, especially that the last one is not responding with `migration_in_progress`.

Reply:
This isn't my spec at all. None of my code changes deal with this part of the code whatsoever. This spec was failing on my last PR #157 because the slack-ruby-client was selecting rtm.connect where it used to call rtm.start and there were no fixtures for that method. I updated the fixture file in the last PR, but the test failed again on this PR, but only on the build server. Worked on my machine (I know), so I surmised that the client had tried to use rtm.start on the build machine, as it has the ability to choose either now. So to handle that I simply duplicated the fixture responses to handle either rtm.start or rtm.connect. 

If that's not the approach you want to take, I'm open to suggestions.

Reply:
That's fine, but the last request that doesn't return migration in progress shouldn't be there then. Which spec fails with that?